docs/source/reference/general_functions.rst

```diff
@@ -14,6 +14,7 @@ Working with options
     reset_option
     get_option
     set_option
+    option_context
```

Comment:
I think we should document at `docs/source/user_guide/options.rst` too. Seems pandas has it too (https://pandas.pydata.org/pandas-docs/stable/user_guide/options.html).

Maybe we can fix some instances at "Operations on different DataFrames" too .. ?

Reply:
There seems one more instance in 10 mins too.

Reply:
I don't think we should add to 10 mins. It's an example for spark conf.